module/metrics/example/verification/main.go

```diff
@@ -148,98 +148,95 @@ func demo() {
 		// This is done to stretch metrics and scatter their pattern
 		// for a clear visualization.
 		for i := 0; i < 100; i++ {
-			// finder
-			if rand.Int()%2 == 0 {
-				verificationCollector.OnExecutionReceiptReceived()
-			}
+			// assigner
+			tryRandomCall(vc.OnExecutionReceiptReceived)
+			tryRandomCall(func() {
+				vc.OnChunksAssigned(rand.Int() % 10)
+			})
+			tryRandomCall(vc.OnChunkProcessed)
+			tryRandomCall(vc.OnFinalizedBlockReceived)
 
-			if rand.Int()%2 == 0 {
-				verificationCollector.OnExecutionResultSent()
-			}
+			// finder
+			tryRandomCall(vc.OnExecutionReceiptReceived)
+			tryRandomCall(vc.OnExecutionResultSent)
```

Comment:
these two are meant to be removed once `assigner` replaces `finder`, though this file is just a working example of VN metrics (not production code).